get_CLN_data.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+chain=$1
```

Comment:
$chain is not used in the script

Reply:
removed by c7b5bd51b8e5520fdc92bb0097acedf04667ae56